modules/special-rules/pages/assault-1.adoc

```diff
@@ -0,0 +1,9 @@
+= Assault +1
+
+These units are particularly fierce melee fighters, without sacrificing their ranged capability -- such as Space Wolves Long Fangs or Grey Hunters.
```

Comment:
```suggestion
These units are exceptionally fierce or gifted at close combat; yet they are no less capable or less equipped for ranged combat than others of their kind.
```
I don't think the examples are essential; and if we drop them then the allusion to 'kind' ('base unit' in terms of the rules) works better.

Reply:
OK. This was basically a shout-out to 3rd edition players which can be replaced with an edition-change note instead.

Reply:
Ah, I see — yes, we'll want to add an 'edition change note' anyway, so it seems like a good idea to put such examples in there as an aid for 3rd edition players. Nice, I'm in favour.